core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -83,6 +83,125 @@ wasm_unload(WASMModule *module)
     wasm_loader_unload(module);
 }
 
+bool
+wasm_resolve_symbols(WASMModule *module)
+{
+    bool ret = true;
+    uint32 idx;
+    for (idx = 0; idx < module->import_function_count; ++idx) {
+        WASMFunctionImport *import = &module->import_functions[idx].u.function;
+        bool linked = import->func_ptr_linked;
+#if WASM_ENABLE_MULTI_MODULE != 0
+        if (import->import_func_linked) {
+            linked = true;
+        }
+#endif
+        if (!linked && !wasm_resolve_import_func(module, import)) {
+            ret = false;
+        }
+    }
+    return ret;
+}
+
+#if WASM_ENABLE_MULTI_MODULE != 0
+static WASMFunction *
+wasm_runtime_resolve_function(const char *module_name,
```

Comment:
Had better change to `wasm_resolve_function`, function `wasm_runtime_xxx` is normally a `core/iwasm/common` layer function for both interpreter and aot. `wasm_xxx` is normally for interpreter.

Reply:
Makes sense.